channels/email/filters.rst

```diff
@@ -17,33 +17,54 @@ will be applied to the ticket that is created or updated by this email.
 
 Zammad comes with system filters as well. While you can't change them, it might
 be useful for you what they actually do. Learn more on
-:doc:`filters/system-filters`
+:doc:`filters/system-filters`.
+
+Different attributes of a filter can be combined with each other. Likewise,
+the following actions can be combined. The supported matches are:
+
+   * Contains
+   * Contains not
+   * Is any of
+   * Is none of
+   * Starts with one of
+   * Ends with one of
+   * Matches regex
+   * Does not match regex
```

Comment:
I recommend we style the list of actual operators like in other places: lowercase and in *italics*. Something like this:

```rst
   * *contains*
   * *contains not*
   * *is any of*
   * *is none of*
   * ...
```